addons/mail/static/src/components/composer_suggestion/composer_suggestion.xml

```diff
@@ -3,27 +3,27 @@
 
     <t t-name="mail.ComposerSuggestion" owl="1">
         <t t-if="record">
-            <a class="o_ComposerSuggestion dropdown-item" t-att-class="{ 'active': props.isActive }" t-attf-class="{{ className }}" href="#" t-att-title="title()" role="menuitem" t-on-click="_onClick" t-ref="root">
+            <a class="o_ComposerSuggestion dropdown-item d-flex w-100 py-2 px-4" t-att-class="{ 'active': props.isActive }" t-attf-class="{{ className }}" href="#" t-att-title="title()" role="menuitem" t-on-click="_onClick" t-ref="root">
                 <t t-if="isCannedResponse">
-                    <span class="o_ComposerSuggestion_part1 text-truncate"><t t-esc="record.source"/></span>
-                    <span class="o_ComposerSuggestion_part2 text-truncate"><t t-esc="record.substitution"/></span>
+                    <strong class="o_ComposerSuggestion_part1 flex-shrink-0 mw-100 pr-2 text-truncate"><t t-esc="record.source"/></strong>
```

Comment:
it should be `pe` instead of `pr`. SCSS was `padding-inline-end`, "end" is important for proper support in both LTR and RTL.

Reply:
Indeed, but when I look at the BS5 documentation, it seems that `pe-x` and `ps-x` always use the `padding-left` and `padding-right` properties.
Depending on the writing mode, BS5 uses the right properties between `padding-left` and `padding-right` :
https://getbootstrap.com/docs/5.1/utilities/spacing/#:~:text=e%20%2D%20(end)%20for%20classes%20that%20set%20margin%2Dright%20or%20padding%2Dright%20in%20LTR%2C%20margin%2Dleft%20or%20padding%2Dleft%20in%20RTL

But when I test with the current `pr-2`, this system already exists and switches to a `padding-left` when we are in RTL mode.
 
![Capture d’écran 2022-03-31 à 11 35 10](https://user-images.githubusercontent.com/80679690/161026039-658812af-f981-4b43-b6da-cdffae923160.png)


So for me, there are two solutions:
- Keep the `pr-2` as it is.
- Keep the `padding-inline-end` in raw css and replace it with the `pe-2` class after the migration.

What do you think and what do you prefer? 🙂

Reply:
We currently have a pre-processing tool that converts all padding-left to padding-right in rtl mode (named `rtlcss`). If you don't have that installed locally it won't work as expected.

But the ideal moving foward would be to use native start and end, and get rid of this tool some day (it might do more than just changing the direction of spaces, I'm not sure exactly on what we rely).

Reply:
To avoid any conflict, I put the padding back in the raw css.
What do you think about it? @seb-odoo @alexkuhn

Reply:
I'm not sure about the start/end issue in bootstrap specifically, maybe check with @stefanorigano he would probably know more about it. I think we already had some usage of that in other PR.

Reply:
@seb-odoo @alexkuhn @stefanorigano 
I put the class "pr-2" back 👍